working-groups/community_meeting/26-01-2023.md

```diff
@@ -0,0 +1,47 @@
+# 26-01-2023 [community]  
+## Agenda - Open to Discussion
+- GSoC & GSoD participation
+- Publications about moja global projects
+- Participation in conferences/meetups
+- Implementation of metrics to track progress
+
+## Discussion
+
+- Creating Blogs for the moja global organization. Like the technology on which we are working like Vuejs for frontend and all.
+- Example - Harsh is taking the blog on “DevOps practises” and “FLINT.Cloud GCP” by Gopi 
+- You being recognized by the name and able to learn lot’s of thing regarding the MojaGlobal just by writing about the same.
+- Platform to write Blogs: Medium 
```

Comment:
Can we provide Medium platform link as well?

Reply:
Publications will be published via either our personal accounts or a collective medium page of Moja Global. @HarshCasper can give more clarity regarding this.